doc/rst/tools/chpl-language-server/chpl-language-server.rst

```diff
@@ -62,6 +62,39 @@ VSCode
 Install the ``chapel`` extension from the `Visual Studio Code marketplace
 <https://marketplace.visualstudio.com/items?itemName=chpl-hpe.chapel-vscode>`_.
 
+Emacs
+^^^^^
+
+With Emacs 29.1, support has been added for language server protocols via `Eglot
+<https://www.gnu.org/software/emacs/manual/html_mono/eglot.html>`_
+
+To utilize the Chapel language server with Eglot, add the following to your
+``.emacs`` file:
+
+.. code-block:: lisp
+
+  (with-eval-after-load 'eglot
+    (add-to-list 'eglot-server-programs
+                 '(chpl-mode . ("chpl-language-server"))))
```

Comment:
I think it's worth noting that this adds chpl-mode from whatever syntax highlight / chapel language "base" support that we have. That is, `chpl-mode` is not built in, and is something we provide as well, i believe.

This stuff:
```Elisp
; Chapel support -- Chapel mode.
(setq chpl-home (getenv "CHPL_HOME"))
(setq chpl-highlight (concat chpl-home "/highlight/emacs/"))
```

Reply:
Right, yeah.  I thought we had some Emacs syntax highlighting documentation, so I'll double check where that is and link to it from here

Reply:
Oof, looks like we just point users to the README in the `$CHPL_HOME/highlight` directory instead of having a page online...

Reply:
That's unfortunate. I suppose that means for the time being it's fair to omit this, but perhaps it's worth opening an issue and moving the `$CHPL_HOME/highlight` docs into `docs` the way we did with CLS and chplcheck.

Reply:
I'll open an issue to talk about moving that documentation online as well, but I think it should be a separate effort

Reply:
Heh, jinx XD